packages/server/controllers/guilds.js

```diff
@@ -114,6 +114,16 @@ async function getLeaderboard(guildId) {
   return members;
 }
 
+async function isGuildMember(guildId, userId) {
+  const guildMember = await prisma.guildMembers.findFirst({
+    where: {
+      guildId: guildId,
+      userId: userId,
+    },
+  });
```

Comment:
use `findUnique()` instead of `findFirst()`, we're trying to query by the guild members table's unique identifier, the `guildId` and `userId`.